commands/fetch_templates.go

```diff
@@ -4,263 +4,171 @@
 package commands
 
 import (
-	"archive/zip"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"log"
-	"net/http"
 	"os"
 	"path/filepath"
-	"strings"
-	"time"
 
-	"github.com/openfaas/faas-cli/proxy"
+	"github.com/openfaas/faas-cli/vcs"
```

Comment:
What does `vcs` mean?

Reply:
Version Control System

Reply:
My point is that it's an abbreviation - could we use something more obvious? We are only using one VCS - git.. could that work?

"github.com/openfaas/faas-cli/git"

or

"github.com/openfaas/faas-cli/versioncontrol"

Maybe?

"github.com/openfaas/faas-cli/template"

Don't change it - but I do want your thoughts.

Also is the code original or is there copy/paste from the Go source code? What is the licensing implication if we copy the code?

Reply:
VCS is a pretty common abbreviation used for version control system, also dvcs for distributed systems.  See for an example of usage https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control 

I do agree that the more verbose `versioncontrol` is  clearer.

Beyond naming, I will rewrite the code to make it more particular to openfaas and not a copy from go.